### ISSUE TITLE
Meeting for Chemistry Writing Style Syncing for Characterisation Data

### ISSUE THREAD

**Issue by fantasy121** [OpenSourceMycetoma/Series-1-Fenarimols#75]
@kym834 @dmitrij176 @fantasy121 

Setting up a meeting for writing style. Please put your availability below.

**Comment by fantasy121** [comment]
Time converter for Sydney + London [here](https://www.timeanddate.com/worldclock/meetingtime.html?year=2022&month=4&day=30&p1=240&p2=136)
<img width="680" alt="Screen Shot 2022-04-27 at 7 09 12 am" src="https://user-images.githubusercontent.com/36176821/165392914-dabfbd84-6aab-497f-925c-a4322cdfa5eb.png">
<img width="672" alt="Screen Shot 2022-04-27 at 7 09 00 am" src="https://user-images.githubusercontent.com/36176821/165392932-644768e9-a81e-4d1b-a050-ad2cbe7c3d3e.png">

**Comment by fantasy121** [comment]
I found some good time slots for the meeting. Looks like it's similar time to what we have for monthly meetings. I'm happy to do Thursday or Friday this week if you're free as well. (We can also potentially do the morning Sydney hour and evening London hour version as well (like our Apr meeting just then) if you want.

**Comment by kym834** [comment]
Thursday 5 PM Sydney and 8 AM London works for me.

**Comment by dmitrij176** [comment]
Hi guys. Any chance to do it on Saturday or Monday?

**Comment by dmitrij176** [comment]
By the way, for how long in terms of time are we aiming?

**Comment by kym834** [comment]
I can do Monday, busy on Sat though. I think we will only need 30 mins.

**Comment by dmitrij176** [comment]
> I can do Monday, busy on Sat though. I think we will only need 30 mins.

Lets do Monday. @fantasy121 are you ok with that day?
What time @kym834 ?

**Comment by kym834** [comment]
@dmitrij176 5 PM Sydney and 8 AM London?

**Comment by dmitrij176** [comment]
> @dmitrij176 5 PM Sydney and 8 AM London?

Ok

**Comment by kym834** [comment]
A zoom link for the meeting https://uni-sydney.zoom.us/j/85031777919?from=addon

**Comment by dmitrij176** [comment]
@kym834 @fantasy121 guys, can we do it 6 pm Sydney and 9 am London time instead ?

**Comment by kym834** [comment]
If we want to push back, it is better at 7PM Sydney and 10AM London for me.

**Comment by dmitrij176** [comment]
> If we want to push back, it is better at 7PM Sydney and 10AM London for me.

Great, thanks a lot.

**Comment by dmitrij176** [comment]
Of course if Hung is ok with that as well.

**Comment by fantasy121** [comment]
That’s good with me too

**Comment by mattodd** [comment]
Hi all - any conclusions here about ways forward or things needing resolving?
Please do remember the thing about needing to compile all the raw NMR data as supporting information for the paper (i.e. not just PDFs of novel compounds, but the NMR data themselves) - will save time later if these are e.g. already on lab notebooks so that they can be collected together and submitted (most likely to a uni repository, given the likely size).

**Comment by kym834** [comment]
@mattodd yes. We've chatted through what we want to do to make all the experimental write up consistent so that we all are following the same formatting. We went through the C-F coupling as well and @dmitrij176 and @fantasy121 are going to update the <sup>13</sup>C NMR data along with the rest of the experimental to reflect these and the things below once we made some decision on them.

We know that often people have preferences for this type of stuff so there are a few points we wanted to discuss with you, @alintheopen and Peter (forgotten his handle again).

* Firstly about purity - because we've tested compounds biologically, I think we should really be including purify at least for those which have gone through to in vivo but really should be checking it for them all even if we don't include it in the SI. @dmitrij176 LRMS has been done using LCMS so we can get purity from that for all the compound @dmitrij176 has contributed. @fantasy121 does not have this so if we decide we want to include it, we'll need to organise for @fantasy121 compounds (and my own) to be ran on the LCMS or HPLC.
* The next is about abbr. we use - do we want to write out the names e.g. acetonitrile or use an abbr. ACN or MeCN. Similar for DCM or CH<sub>2</sub>Cl<sub>2</sub> etc? Do we want to us rt or write out room temperature?
* Do we want to include <sup>19</sup>F NMR? - @fantasy121 and I have this data already, @dmitrij176 would need to collect it. Note we do not have a reference solvent or standard that we have included in these experiments.
* For <sup>13</sup>C NMR - report to 1 or 2 decimal places?
* As we have Cl and Br in some compounds we have isotopes in the MS - do we want to include for example both the <sup>35</sup>Cl and the <sup>37</sup>Cl molecules when reporting HRMS?

This is more related to the whole paper - @wwjvdsande and @Wilson-Lm 
* References - what system do we want to use. @dmitrij176 uses EndNote, @fantasy121 and I use Zotero. This doesn't matter too much and maybe rather what we need to decide is who will be taking responsibility for the referencing and then everyone can send them their references and up to the that person to put it all in and format it.

As for where to collate the raw data - we could add it to another folder in the DropBox, into a folder here within this GitHub repo or on GDrive I guess. I personally include my raw data files on GitHub in the ELN repo so it's all in one place so anyone can easily grab mine from there and I think it works quite well - you can check it out [here](https://github.com/TheBreakingGoodProject/ELN-Kymberley-Scroggie/tree/master/Characterisation%20Data) if you like and see if you think some similar would work for this.

**Comment by dmitrij176** [comment]
@kym834 thanks for the update.

**Comment by mattodd** [comment]
OK, thanks @kym834. My take:

1) Purity. Certainly expected for anything biologically evaluated. LCMS can be used, as can copies of 1H and 13C NMR spectra of the evaluated sample.
2) Abbreviations, yes just decide on a set and make a list up front.
3) 19F is nice if you have it, and would be nice for any novel compounds, but the ppm value will not change much for these compounds so it's of limited value in terms of characterisation.
4) 13C, I'd say 1 d.p. If majority already 2 d.p. then use that.
5) References - we can do that at the end. You can use DOIs as the markers for now if that helps.

Yes, I would ensure that each compound has the raw data in the ELN entry (this is the best way to do things anyway) and then at the last moment we collate and upload to a uni repo, installing the relevant link in the paper. Ditto with the ELNs, really - we want to have Labarchives "offline copies" of the relevant ELNs (so, not PDF versions) so that people can browse them.

Do you have a representative write-up that you could share where everyone is pretty happy with how it's done, so we could comment on anything else arising?

**Comment by kym834** [comment]
@mattodd I've copied over the first few procedures from the paper to this [google doc](https://docs.google.com/document/d/1jUMTSB2543FK6vYs_yfP6IHhUAWAyWo2pXS7ywYvDQk/edit?usp=sharing) for us to work on together to get the formatting right. I've updated these to the formatting that we have decided on already (🤞 I didn't miss anything). Should be open for everyone to edit.

Addressing other points:
1. For purity, would be good to do purity from NMR as this will save time. Do you have a preferred method? What I've used before you need to know the amount of sample and I don't know if we have the info for the spectra we already have @fantasy121 @dmitrij176?? 
2. Okay - we can work on that between ourselves and make sure it is consistent throughout.
3. I agree. It's a nice add to show that there is indeed a F but not really useful beyond that. As @fantasy121 and I already have F NMR I'll leave the decision on if we include it or not up to others. I'd have a preference to have it at least for novel compounds.
4. 👍 Majority are 1 d.p so we'll stick with that.
5. @fantasy121 @dmitrij176 if you can add in the DOI's for referencing later - that would be fab!

**Comment by kym834** [comment]
@dmitrij176 and @fantasy121 let's get started on pulling together the experimental for those compounds which have been tested _in vivo_. This will help Wilson to get his PhD submitted, we can share with Mat and others to finalised the formatting and then use all this info to work on all the other compounds. 

By my count (and I've as Wilson to double check, and maybe you both can to) these are the compounds which have already been tested in vivo which we'll need the experimental and purity for 
https://docs.google.com/spreadsheets/d/1YhK-3i2KwuVabo1GbZSgVjAUbavEICCMKi5v-EhNq80/edit#gid=441422947. A number are from the Epichem library so less work for us there :)

I've created a [new word document](https://www.dropbox.com/s/8kcvv280hlgyty4/Experimental%20for%20In%20Vivo%20Compounds.docx?dl=0) in the Dropbox for us to work on this. Once we get it all in and everyone is happy with it we can copy over to the draft.

**Comment by Wilson-Lm** [comment]
Hi all, 
I've confirmed the number of compounds that were tested _in vivo_. The ones listed in the excel sheet (22 in total) is correct. 
Thanks for compiling this @kym834  

Looking forward to see the latest version :)
